src/sc/sidechainrpc.cpp

```diff
@@ -885,11 +885,10 @@ void ScRpcCmd::addInputs()
 
     static const bool fOnlyConfirmed = false;
     static const bool fIncludeZeroValue = false;
-    bool fProtectCoinbase = !Params().GetConsensus().fCoinbaseMustBeProtected;
-    static const bool fIncludeCoinBase = fProtectCoinbase;
-    static const bool fIncludeCommunityFund = fProtectCoinbase;
+    const bool includeCoinBase = !ForkManager::getInstance().mustCoinBaseBeShielded(chainActive.Height() + 1);
+    const bool includeCommunityFund = ForkManager::getInstance().canSendCommunityFundsToTransparentAddress(chainActive.Height() + 1);
 
-    pwalletMain->AvailableCoins(vAvailableCoins, fOnlyConfirmed, NULL, fIncludeZeroValue, fIncludeCoinBase, fIncludeCommunityFund);
+    pwalletMain->AvailableCoins(vAvailableCoins, fOnlyConfirmed, NULL, fIncludeZeroValue, includeCoinBase, includeCommunityFund);
```

Comment:
I may be missing something here, but wasn't the goal to restore the initial variable names here (meaning `fIncludeCoinBase` and `fIncludeCommunityFund`)? It's not a big deal obviously, just a matter of style consistency.

Reply:
Yes, I renamed from `fMustShieldX` to `includeX`, forgetting about the prepended `f`; adapted accordingly with `fIncludeCoinBase` and `fIncludeCommunityFund`.